Documentation/ceph-upgrade.md

```diff
@@ -471,6 +463,17 @@ It's best to run `ceph config-key dump` again to verify references to
 
 See for more information, see here: https://github.com/rook/rook/issues/9185
 
+### **Rename CephBlockPool device_health_metrics pool when upgrading to Quincy v17**
+In Ceph Quincy (v17), the `device_health_metrics` pool was renamed to `.mgr`. Ceph will perform this
+migration automatically. If you do not use CephBlockPool to customize the configuration of the
+`device_health_metrics` pool, you don't need to do anything further here.
+
+If you do use CephBlockPool to customize the configuration of the `device_health_metrics` pool, you
+will need to do a few steps after the Ceph upgrade is complete. Once upgrade is complete, create a
+new CephBlockPool to configure the `.mgr` built-in pool. You can reference the example
+[builtin mgr pool](https://github.com/rook/rook/blob/{{ branchName }}/deploy/examples/pool-builtin-mgr.yaml).
```

Comment:
Shall we have them check if the pool was renamed successfully? Or did ceph convert it seemlessly in your upgrade testing?